src/Raven.Server/Documents/Handlers/Streaming/StreamingHandler.cs

```diff
@@ -158,6 +158,9 @@ public async Task StreamQueryGet()
                 }
                 var query = await IndexQueryServerSide.CreateAsync(HttpContext, GetStart(), GetPageSize(), queryContext.Documents, tracker, overrideQuery: overrideQuery);
                 query.IsStream = true;
+                
+                if (query.Metadata.Includes != null)
+                    throw new NotSupportedException("Includes are not supported by streaming query.");
```

Comment:
You're throwing earlier now - that's fine. Although you should remove the code which currently throws because after this change it's effectively dead code.

Reply:
Removed it in query internal